### ISSUE TITLE
Calling methods on nodes returned by CypherQuery is awkward

### ISSUE THREAD

**Issue by nieware** [jmcvetta/neoism#60]
I was stuck for a few hours today trying to figure out what I was doing wrong until the helpful guys at StackOverflow pointed me in the right direction. See here for details and example code:

http://stackoverflow.com/questions/27405705/golang-neoism-calling-methods-of-objects-returned-by-query-leads-to-panic

It turns out that you have to initialize the Db property of nodes returned by CypherQuery manually before you can call methods like SetProperty. Of course, you can do that once you are aware it has to be done, but it feels very awkward. Is there a more intuitive way of handling this?

**Comment by wwwdata** [comment]
hm, got this problem today too. I was querying for a result struct with a node in it, see:

```
res := []struct {
		Node *neoism.Node
	}{}
```

so far so good, got the node with all the data, but calling `SetProperties` returned a 

```
runtime error: invalid memory address or nil pointer dereference
```

after setting `res[0].Node.Db = myDbObject` it worked ... This should be filled automatically by neoism

**Comment by jmcvetta** [comment]
I agree, it would be better if Neoism could set this variable automagically.  However please see my comment above as to why doing this is decidedly non-trivial.

If you would like to take a shot implementing this behavior, I will be very happy to accept the Pull Request.

**Comment by dowlingw** [comment]
+1 also hit this today, the documentation example returning a Node object is misleading.
At the very least clarifying this in the docs would be great.

Otherwise thanks for a really useful lib! :)

**Comment by jmcvetta** [comment]
Glad you enjoy Neoism!

Please feel free to update the docs in a way you think would clarify the current behavior and make things easier for other users.  I will be happy to accept the PR.  :)
